Index: djangotest/main/mqttoutput.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from .__init__ import connect\r\n\r\n\r\nclass blink:\r\n    def __init__(self):\r\n        blink.self = self\r\n\r\n\r\n    def blinkOn():\r\n        client = __init__.client()\r\n        client.publish(topic, 1)
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/djangotest/main/mqttoutput.py b/djangotest/main/mqttoutput.py
--- a/djangotest/main/mqttoutput.py	(revision 2c48210c3e3555edab65dea68079a6502a90c457)
+++ b/djangotest/main/mqttoutput.py	(date 1649677812232)
@@ -6,6 +6,6 @@
         blink.self = self
 
 
-    def blinkOn():
-        client = __init__.client()
+    def blinkOn(self):
+        client = connect.cl()
         client.publish(topic, 1)
\ No newline at end of file
Index: djangotest/main/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from paho.mqtt import client as mqtt_client\r\n\r\n\r\nbroker = '192.168.4.1'\r\nport = 1883\r\ntopic = \"test/blink\"\r\nclient_id = f'python-mqtt-{0}'\r\n\r\n\r\nclass connect():\r\n    def __init__(self):\r\n        connect.self = self\r\n\r\n    def connect_mqtt(broker, port, topic, client_id):\r\n        def on_connect(client, userdata, flags, rc):\r\n            if rc == 0:\r\n                print(\"Connected to MQTT Broker!\")\r\n            else:\r\n                print(\"Failed to connect, return code %d\\n\", rc)\r\n        # Set Connecting Client ID\r\n        client = mqtt_client.Client(client_id)\r\n        #client.username_pw_set(username, password)\r\n        client.on_connect = on_connect\r\n        client.connect(broker, port)\r\n        return client\r\n\r\n    def client(self):\r\n        return client\r\n\r\n\r\nclient = connect.connect_mqtt(broker, port, topic, client_id)\r\nclient.loop_start()
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/djangotest/main/__init__.py b/djangotest/main/__init__.py
--- a/djangotest/main/__init__.py	(revision 2c48210c3e3555edab65dea68079a6502a90c457)
+++ b/djangotest/main/__init__.py	(date 1649677812232)
@@ -24,7 +24,7 @@
         client.connect(broker, port)
         return client
 
-    def client(self):
+    def cl(self):
         return client
 
 
